change/@ni-nimble-components-e5bfd621-8ece-4d9c-a6ea-eb128a70e8ac.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Seems like a pre-exisiting issue but the keyboard behavior when in the initial state of no selected value is broken. Can't tab to the control at all.

Reply:
Turns out that this is an issue with our storybook example, not the component itself. We were setting the value to an invalid option (the string "none") rather than setting the value to `undefined`. I'll fix that as part of this PR.